packages/edge-login-ui-rn/src/components/themed/EdgeTextField.js

```diff
@@ -14,10 +29,27 @@ class EdgeTextFieldComponent extends React.PureComponent<ThemeProps> {
         tintColor={theme.primaryText}
         baseColor={theme.primaryText}
         errorColor={theme.dangerText}
+        containerStyle={spacingStyles(props, theme)}
         {...props}
       />
     )
   }
 }
 
 export const EdgeTextField = withTheme(EdgeTextFieldComponent)
+
+function spacingStyles(props: Props, theme: Theme) {
+  const marginRem = unpackEdges(props.marginRem)
+  const paddingRem = unpackEdges(props.paddingRem)
+
+  return {
+    marginBottom: theme.rem(marginRem.bottom),
+    marginLeft: theme.rem(marginRem.left),
+    marginRight: theme.rem(marginRem.right),
+    marginTop: theme.rem(marginRem.top),
+    paddingBottom: theme.rem(paddingRem.bottom),
```

Comment:
We can just delete all this padding stuff, since it's not needed.

What is left over should be short enough to just smash inline into `EdgeTextFieldComponent.render` itself.